README.md

```diff
@@ -180,6 +181,9 @@ The PAPR was designed and tested using the following setup.  You may need to mak
 - This allows the supports to come in at an angle that makes them easier to remove.
 - This is the maximum angle that still allows the supports needed to correctly print the bayonet teeth in the inlet.
 - This orientation preserves the quality of the printed text.
+- Most printing processes will require the 6-32 treads the body be chased with a 6-32 tap.  I recommend doing this step before post-curing, while the resin is still soft.
```

Comment:
First sentence seems awkward. Also, could add the tap to the tools list:
https://www.mcmaster.com/25995A337/
or maybe this:
https://www.mcmaster.com/2529A46/
?
Also "treads"

Reply:
These changes are in

Reply:
Added amazon tap tool list because full kit is about the same price as mcmaster single tap + handle